src/LibExpat.jl

```diff
@@ -15,7 +15,6 @@ include("lX_defines_h.jl")
 include("lX_expat_h.jl")
 #include("lX_exports_h.jl")
 
-@c Ptr{XML_LChar} XML_ErrorString (Cint,) libexpat
```

Comment:
It's already defined in https://github.com/amitmurthy/LibExpat.jl/blob/882d8fc8e8ac899aaffa85b8022a849651c6fd77/src/lX_expat_h.jl#L66, so it produces a method redefinition warning. However, upon further inspection, maybe the other method needs to be changed to take `Cint` instead of `Void`?